coursebuilder/modules/explorer/_static/components/pages/homepage.html

```diff
@@ -27,22 +25,10 @@ <h1 class="intro-title">Improve your digital storytelling with a combination of
       <!--  Mission  -->
       <div class="section mission blue-section">
         <div class="blue-section-inner">
-          <h2>What Is The StoryLab Academy?</h2>
-          <p>
-            What is the StoryLab Academy: Journalism is evolving so quickly that newsrooms are
-            struggling to keep up.</p>
-          <p>
-            The StoryLab Academy seeks to help, by bringing face-to-face training into partner
-            newsrooms across Africa, and by hosting public workshops at monthly Hacks/Hackers
-            meetups.
-          </p>
-          <p>
-            The Academy will also offer online courses and webinars, designed to teach just one tool
-            or technique at a time, so participants can upgrade their skills at their own pace.
-          </p>
+          <h2>Learn on the Go</h2>
           <p>
-            The training will be spearheaded by the continent’s largest digital journalism network,
-            Code for Africa, with support from the Google News Lab and World Bank.
+          As a journalist living in the digital age, a huge demand falls on you to be up to date with the latest in data gathering and processing skills.These courses are tailored to help you do just that.
+          From learning Google apps that can make your investigative work easier to delving deeper into the intricacies of extracting and understanding data. You will become a head honcho in the latest techniques used in news gathering.
```

Comment:
@Sharonsyra <br/> ?

Reply:
Muthoni added new data that i was supposed to update. It is in the word document.

Reply:
Did you intend to have a line break between "just that." and 
"From learning" ?

Reply:
Aah! Thanks let me add one.